diff -Naur Vulkan-Loader-1.3.243/loader/vulkan.pc.in Vulkan-Loader-1.3.243.tpg/loader/vulkan.pc.in
--- Vulkan-Loader-1.3.243/loader/vulkan.pc.in	2023-03-13 19:18:04.000000000 +0000
+++ Vulkan-Loader-1.3.243.tpg/loader/vulkan.pc.in	2023-03-16 15:56:38.576574020 +0000
@@ -6,6 +6,6 @@
 Name: @VULKAN_LOADER_PKG_CONFIG_NAME@
 Description: Vulkan Loader
 Version: @VULKAN_LOADER_VERSION@
-Libs: -L${libdir} -lvulkan@VULKAN_LIB_SUFFIX@
+Libs: -lvulkan
 @PRIVATE_LIBS@
 Cflags: -I${includedir}
