diff -uraN Vulkan-Loader-1.2.166/loader/vulkan.pc.in omv-Vulkan-Loader-1.2.166/loader/vulkan.pc.in
--- Vulkan-Loader-1.2.166/loader/vulkan.pc.in	2021-01-05 18:45:38.000000000 +0100
+++ omv-Vulkan-Loader-1.2.166/loader/vulkan.pc.in	2021-01-12 14:55:14.029957279 +0100
@@ -6,7 +6,7 @@
 Name: @CMAKE_PROJECT_NAME@
 Description: Vulkan Loader
 Version: @VK_API_VERSION@
-Libs: -L${libdir} -lvulkan@VULKAN_LIB_SUFFIX@
+Libs: -lvulkan
 Libs.private: @PRIVATE_LIBS@
 Cflags: -I${includedir}
 
